widget/src/api/user-session/login.ts

```diff
@@ -110,4 +110,34 @@ export class LoginAPI implements LoginAPICodeChallengeStorageAccessor {
 
     return url.toString();
   }
-}
+
+  createRegisterURL(payload?: string): string {
```

Comment:
I think we don't need to create a new function here, just modify the `createLoginURL`. Rename it to something like this `createLoginWebsiteUrl`, and let the developer set the `target_auth_page` by them self

`createLoginWebsiteUrl({ payload: {}, targetAuthPage: "register" })`